File: Flutter/app_exercicio/lib/src/models/bebida.dart
class Bebida {
  final int id;
  final String descricao;
  final String categoria;
  final String imgUrl;
  final double valor;

  Bebida({
    required this.id,
    required this.descricao,
    required this.categoria,
    required this.imgUrl,
    required this.valor,
  });

  factory Bebida.fromJson(Map<String, dynamic> json) {
    return Bebida(
      id: json['id'],
      descricao: json['descricao'],
      categoria: json['categoria'],
      imgUrl: json['imgUrl'],
      valor: json['valor'].toDouble(),
    );
  }
}

File: crud-elias/crud/src/main/java/com/flutter/crud/entities/Bebida.java
package com.flutter.crud.entities;
import jakarta.persistence.*;

@Entity
public class Bebida {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;
    
    @Column(nullable = false)
    private String descricao;
    
    @Column(nullable = false)
    private String categoria;
    
    @Column(nullable = false)
    private String imgUrl;
    
    @Column(nullable = false)
    private double valor;
    
}